gui/tests/__init__.py

```diff
@@ -1,3 +1,9 @@
+import os
+import sys
+
 from PySide2.QtWidgets import QApplication
 
+if sys.platform != "darwin":
```

Comment:
this would be for linux && windows?  AIUI, this should only be for linux

Reply:
hmm, good point. We don't have gui tests on Windows in CI (maybe we should?) but regardless, I'll change this to be linux-specific instead.